src/components/Picklist/readme.md

```diff
@@ -317,3 +317,104 @@ initialState = { value: { name: 'option 3', label: 'Central Park' } };
+import React from 'react';
+import { Picklist, Option, Modal, Input, Button, GoogleAddressLookup } from 'react-rainbow-components';
+import { Field, reduxForm } from 'redux-form';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBuilding } from '@fortawesome/free-solid-svg-icons';
+import styled from 'styled-components';
+
+initialState = { isOpen: false, selection: { name: '', label: '' }, googleAddressValue: null, buildingNameValue: '' };
+
+const StyledInput = styled(Input)`
+    margin-bottom: 90px;
+    margin-top: 32px;
+`;
+
+const containerStyles = {
+    width: '220px'
+};
+
+function handlePicklistChange(value) {
+    if (value.name === 'option 1') {
+        setState({
+            isOpen: true,
+        });
+    } else {
+        setState({ selection: value });
+    }
+}
+
+<div className="rainbow-m-bottom_xx-large rainbow-p-bottom_xx-large">
+    <GlobalHeader
+        src="images/user/user3.jpg"
+        className="rainbow-p-bottom_xx-large rainbow-m-bottom_xx-large"
+        variant="neutral">
+        <div className="rainbow-flex rainbow-align_right">
+            <Picklist
+                id="picklist-15"
+                style={containerStyles}
+                onChange={handlePicklistChange}
+                value={state.selection}
+                placeholder="Select your building"
+                variant="shaded"
+            >
+                <Option name="option 1" label="New Building" icon={<AddFilledIcon />} />
+                <Option name="header" label="Your Buildings" variant="header" />
+                <Option name="option 2" label="Central Park Tower" />
+                <Option name="option 3" label="Empire State" />
+                <Option name="option 5" label="Panorama Tower" />
+                <Option name="option 4" label="Chrysler" />
+                <Option name="option 6" label="Bennet Towers" />
+            </Picklist>
+        </div>
+    </GlobalHeader>
+    <Modal
+        title="New Building"
+        isOpen={state.isOpen}
+        onRequestClose={() => setState({ isOpen: false })}
+        footer={
+            <div className="rainbow-flex rainbow-justify_end">
+                <Button
+                    form="redux-form-id"
+                    className="rainbow-m-right_large"
+                    label="Cancel"
+                    variant="neutral"
+                    onClick={() => setState({ isOpen: false })}
+                />
+                <Button
+                    form="redux-form-id"
+                    label="Save"
+                    variant="brand"
+                    type="submit"
+                />
+            </div>
+        }
+    >
+        <div className="rainbow-p-horizontal_small">
+            <GoogleAddressLookup
+                id="gaddresslookup-1"
+                label="Building Location"
+                required
+                onChange={value => setState({ value })}
+                value={state.googleAddressValue}
```

Comment:
here you are passing value from the googleAddressValue state but in the onChange it is stored in value state when it must be stored in googleAddressValue